lib/faraday_middleware/rack_compatible.rb

```diff
@@ -80,9 +80,7 @@ def restore_env(rack_env)
     def finalize_response(env, rack_response)
       status, headers, body = rack_response
       body = body.inject { |str, part| str << part }
-      unless headers.is_a?(Faraday::Utils::Headers)
-        headers = Faraday::Utils::Headers.new(headers)
-      end
+      headers = Faraday::Utils::Headers.new(headers) unless headers.is_a?(Faraday::Utils::Headers)
```

Comment:
I'm a bit lost as to why the standard 80 chars limit doesn't apply anymore. Is that because of rubocop-packaging?

Reply:
Oh no, it's not related to that at all!
RuboCop has bumped the minimum characters to 120 by default. Hope that's okay?

Reply:
Particularly, from RuboCop 0.84, the default maximum line length has been changed to 120.
cf: https://docs.rubocop.org/rubocop/0.91/cops_layout.html#layoutlinelength

Reply:
I missed that one, but I completely agree with that! 80 chars in 2020 were a bit too restrictive 😄